src/iohook.cc

```diff
@@ -181,10 +181,10 @@ int hook_enable() {
   pthread_attr_getschedpolicy(&hook_thread_attr, &policy);
   int priority = sched_get_priority_max(policy);
   #endif
-  
+
   #if defined(_WIN32)
   DWORD hook_thread_id;
-  DWORD *hook_thread_status = malloc(sizeof(DWORD));
+  DWORD *hook_thread_status = (DWORD *) malloc(sizeof(DWORD));
```

Comment:
This is the only actual change here, the rest is my editor removing whitespace at end of lines.